qlty-cli/tests/cmd/check/diff_tests/diff_mode.stdout

```diff
@@ -1,6 +1,6 @@
+[..]JOBS: 1[..]
 
- ISSUES: 1 
-
-:0:0
-    0:0  high    exists failed  exists:fail
+Plugin  Result   Targets   Time   Debug File
+exists  Success  1 target[..]
 
+Checked 1 modified file
```

Comment:
If I am reading this diff correctly, this seems to be saying that prior to this PR, this integration test would generate a single issue "exists failed", and after this PR, it produces no issues.

Is that expected?

Reply:
Yes. The approach was changed such that we're no longer testing a lint failure and instead using --verbose to log how many targets were linted (which is what we care about here).